src/com/serotonin/mango/rt/maint/work/EmailWorkItem.java

```diff
@@ -90,15 +98,24 @@ public void execute() {
             emailSender.send(fromAddress, toAddresses, subject, content);
         }
         catch (Exception e) {
+            LOG.error(e);
             String to = "";
             for (InternetAddress addr : toAddresses) {
                 if (to.length() > 0)
                     to += ", ";
                 to += addr.getAddress();
             }
-            SystemEventType.raiseEvent(new SystemEventType(SystemEventType.TYPE_EMAIL_SEND_FAILURE),
-                    System.currentTimeMillis(), false,
-                    new LocalizableMessage("event.email.failure", subject, to, e.getMessage()));
+            Boolean dontCreateEventForEmailError = false;
+            try {
+                dontCreateEventForEmailError = ScadaConfig.getInstance().getBoolean(ScadaConfig.DONT_CREATE_EVETS_FOR_EMAIL_ERROR,false);
```

Comment:
Is it possible to rename this variable , it means, by just put char 'N' :
DONT_CREATE_EVETS_FOR_EMAIL_ERROR - > DONT_CREATE_EVENTS_FOR_EMAIL_ERROR